apps/frontend/src/axios/axios.ts

```diff
@@ -1,17 +1,28 @@
 import axios from 'axios';
+import config from '../config';
+
+interface User {
+  username: string;
+  email: string;
+  token: string;
+  password: string;
+  friends: string[];
+  bio: string;
+  img: string;
+}
 
 const instance = axios.create({
-  baseURL: 'http://localhost:5173',
+  baseURL: config.baseURL,
   timeout: 5000,
   headers: {
     'Content-Type': 'application/json',
   },
 });
 
-const getAccessToken = () => {
+const getAccessToken = (): string | null => {
   const userData = localStorage.getItem('user');
```

Comment:
Move to a new user.tsx file , export. { get, set }. Don't use "localStorage" in this file. The logic of where the user is shouldn't be in "getAccessToken" function

Reply:
are you talking about a user class with get and set functions or just a function like the one i made that gets the user from local storage

Reply:
After transpilation a file :
file with class = object the exports object with function
file with named exports = object that exports function

(There is no benefit for class here, it's more OOP but we like functional programming)